resources/core/charts/gateway/templates/kyma-gateway-certs.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if not .Values.global.environment.gardener }}
+ {{ if not .Values.global.environment.gardener }}
```

Comment:
Why do you update the chart if core is removed from the component list? Shouldn't it be removed?

Reply:
I'm removing it from components list for the new installation, but it is still being used by the old one.